modules/osd-storage-pv-aws-config-account.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/
+
+[id="osd-storage-pv-aws-config-account_{context}"]
+= Configuring the AWS account
+
+Set up your AWS account to prepare EFS for use by {product-title}.
+
+.Procedure
+
+. Log in to the link:https://console.aws.amazon.com/ec2[AWS EC2 Console].
+. Select the region that matches the cluster region.
+. Filter only worker EC2 instances, and select an instance. Note the VPC ID and security group ID. These values are required later in the process.
```

Comment:
Not sure if worker can be replaced (compute?) since this is referring to an AWS entity, but wanted to call this out if it was possible.